.travis.yml

```diff
@@ -1,10 +1,9 @@
 language: php
 php:
-  - 5.3
-  - 5.4
+  - 5.5
```

Comment:
I would still run builds with older still supported PHP versions to be sure to not introduce any regressions.